tests/common/connectors/trending-items/options.ts

```diff
@@ -0,0 +1,222 @@
+import {
+  createRecommendResponse,
+  createSearchClient,
+  createSingleSearchResponse,
+} from '@instantsearch/mocks';
+import { wait } from '@instantsearch/testutils';
+import { screen } from '@testing-library/dom';
+
+import { skippableDescribe } from '../../common';
+
+import type { TrendingItemsConnectorSetup } from '.';
+import type { SetupOptions, TestOptions } from '../../common';
+
+export function createOptionsTests(
+  setup: TrendingItemsConnectorSetup,
+  { act, skippedTests }: Required<TestOptions>
+) {
+  skippableDescribe('options', skippedTests, () => {
+    test('forwards parameters to the client', async () => {
```

Comment:
there's no test that removing the widget removes the parameters too. Does that actually work, or does `getParameters` only work with `future.cleanUpOnDispose`?

Reply:
I looked into it and we missed handling this. There is support for removing recommend parameters at the helper level, but this is missing:
- `helper.removeXXX()` expects string ids but we switched to numerical ids
- the connector's `dispose()` method receives the search state instead of the recommend state
- no recommend connector calls `state.removeXXX()` as of now

I'll create a task for tackling this properly.